src/structures/GuildTemplate.js

```diff
@@ -9,10 +9,6 @@ const DataResolver = require('../util/DataResolver');
  * @extends {Base}
  */
 class GuildTemplate extends Base {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APIGuildTemplate} data The raw data for the template
```

Comment:
With this removal, the only other instance of `APIGuildTemplate` is in the `_patch()` method below. Does this really need to be documented? I noticed other `_patch()` methods aren't documented so if both are to go, we could remove `@external` altogether!